.circleci/check_vault_connection.sh

```diff
@@ -7,12 +7,10 @@ NUM_PINGS="${NUM_PINGS:-15}"
 DELAY="${DELAY:-1}"
 
 for try in $(seq $NUM_PINGS); do
-  curl --cert 'tests/ssl/acra-writer/acra-writer.crt' \
-    --key 'tests/ssl/acra-writer/acra-writer.key' \
-    --cacert 'tests/ssl/ca/ca.crt' \
-    -XGET "https://${TEST_VAULT_HOST}:${TEST_VAULT_PORT}/"
+  curl -XGET "https://${TEST_VAULT_HOST}:${TEST_VAULT_PORT}/"
 
-  if [ "$?" -eq "0" ]; then
+    #60 code is SSL certificate problem - means that we can reach host
+  if [ "$?" -eq "60" ]; then
     exit 0
   else
     sleep ${DELAY}
```

Comment:
It seems a bit unstable for me. For example, when for some reason we will trust that certificate, here we will get unexpected behavior.

This code is just about checking TCP connectivity (so far I understand) so nothing more should affect behavior changes.

I'd suggest this approach:
```
curl -s -k "https://${TEST_VAULT_HOST}:${TEST_VAULT_PORT}" && exit 0 || sleep "$DELAY"
```

Reply:
Thanks @shadinua, but even with two separate vault instances we had such issue, so I decided restructure tests a little bit to mount v1/v2 paths only one time at the start of the tests so I redo these changes and keep using one instance of Vault.